package.json

```diff
@@ -13,17 +13,29 @@
   "files": [
     "dist"
   ],
+  "bin": {
+    "create-beacon-reader-app": "./dist/cli/index.js"
```

Comment:
So the expected usage will look something like:

```
npx @api3dao/services create-beacon-reader-app
```

(maybe there needs to be some extra param for npx to correctly find the binary, but that is a detail)